jpa/org.eclipse.persistence.jpa/src/main/java/org/eclipse/persistence/jpa/JpaEntityManager.java

```diff
@@ -36,7 +36,7 @@
  * @author Gordon Yorke
  */
 
-public interface JpaEntityManager extends jakarta.persistence.EntityManager {
+public interface JpaEntityManager extends jakarta.persistence.EntityManager, AutoCloseable {
```

Comment:
I think, that `AutoCloseable` will be applied/moved later to `jakarta.persistence.EntityManager` as a part of JPA specification change?

Reply:
Yes. The idea is to allow usage (and testing on our end) even in older EclipseLink versions (ie 2.7.x) without a need for new API (where it is hard to predict availability of the binary we would be able to use/integrate)